channels/urbdrc/client/isoch_queue.c

```diff
@@ -69,18 +70,24 @@ static ISOCH_CALLBACK_DATA* isoch_queue_register_data(ISOCH_CALLBACK_QUEUE* queu
 		queue->tail->next = (void*)isoch;
 		isoch->prev = (void*)queue->tail;
 		queue->tail = isoch;
+		isoch = NULL;
```

Comment:
This results in incorrectly returning NULL even though the item was successfully added to the queue